modules/client-configuration/pages/autoinstallation-methods.adoc

```diff
@@ -2,14 +2,25 @@
 = Introduction
 
 
-Kickstart and {ay} configuration files allow you to automate client system installations.
-This is especially useful if you need to install a large number of clients.
+{ay} and Kickstart configuration files allow you to automate client system installations.
+This is useful if you need to install a large number of clients.
 
 For {sle} clients, use {ay}.
-When you have created an {ay} file, you can upload it and manage it with {productname}.
+When you have created an {ay} file, you can upload and manage it the within {productname} {webui}.
 
 For {rhel} clients, use Kickstart.
 Kickstart files are created, modified, and managed within the {productname} {webui}.
 
-The Cobbler installation server allows you to automate bare-metal installations.
-It uses DHCP to access a PXE boot server, and can be used in virtualized environments.
+
+
+== Basic Concept
```

Comment:
I don't like this as a section heading. Can we just delete the title? The basic concept should go at the top of the chapter anyway, I don't think it needs a separate section.